README.md

```diff
@@ -12,6 +12,7 @@ Learn about the capabilities of this extension in our [Reliability Hub](https://
 |------------------------------------------------------------------|---------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------------------------------------------------------------------|
 | `STEADYBIT_EXTENSION_KUBERNETES_CLUSTER_NAME`                    | `kubernetes.clusterName`                    | The name of the kubernetes cluster                                                                                                                                 | yes      |                                                                      |
 | `STEADYBIT_EXTENSION_DISABLE_DISCOVERY_EXCLUDES`                 | `discovery.disableExcludes`                 | Ignore discovery excludes specified by `steadybit.com/discovery-disabled`                                                                                          | false    | `false`                                                              |
+| `STEADYBIT_EXTENSION_DISCOVERY_DISABLE_ARGO_ROLLOUTS`            | `discovery.disableArgoRollouts`             | Disable discovery of Argo rollouts                                                                                                                                 | false    | `true`                                                              |
```

Comment:
The env var is `STEADYBIT_EXTENSION_DISCOVERY_DISABLED_ARGO_ROLLOUTS` (missing `D` in disableD)

(Also the helm value - see my comment in the helm values.yaml)